heka/sandbox/decoders/http_edge_decoder.lua

```diff
@@ -99,6 +102,13 @@ function process_message()
     landfill_msg.Fields.submissionDate = os.date("%Y%m%d", landfill_msg.Timestamp / 1e9)
     landfill_msg.Fields.submission = {value = read_message("Payload"), value_type = 1} -- Bugzilla 1204589
 
+    -- Reopen city_db once an hour.
+    local current_hour = math.floor(os.time() / 3600)
+    if current_hour > hour then
+        city_db:close()
+        city_db = assert(geoip.city.open(read_config("geoip_city_db")))
```

Comment:
How does the new install happen?  i.e., will it always be available at this point? If this open fails the decoder is terminated.